### ISSUE TITLE
Experiment with hyperparameter sensitivity

### ISSUE THREAD

**Issue by CharlizeHanekom** [CharlizeHanekom/Transfer-Learning-for-Emotion-Analysis-in-Bantu-Languages#8]
Learning rate, batch size, dropout, epochs.
